hkm/migrations/0037_auto_20200601_1547.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Just a personal opinion, but we could rename this file to something like `0037_change_deprecated_field_type_for_printer_ip.py`

Reply:
Good point. I renamed the file as you suggested.